osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -552,5 +520,35 @@ private void load(OsuColour colours)
                 }
             }
         }
+
+        public class BeatmapInfoWedgeContainer : Container
```

Comment:
This naming is confusing.

You currently have a `BeatmapInfoWedge` creating a `BeatmapInfoWedgeContainer` which has `WedgeInfoText`. The ordering of these classes is 1-3-2. One doesn't contain the prefix, while the other two do. The "Container" is inside the non-container (which is back to front to how we usually have it) and I don't think the term container suits in this use case.

Reply:
On second glance the `BeatmapInfoWedgeContainer` isn't actually needed. It is not really doing anything to justify to be it's own class. Would be enough to change https://github.com/ppy/osu/pull/12503/files#diff-10508e952755970ee0b1dfc1cfaac15497e2e8153598c73c0133e3a3e367ca5fR137 to a normal `Container` instantiation, right?

Reply:
I think that will help a lot, yes.